dhis-2/dhis-api/src/main/java/org/hisp/dhis/programrule/ProgramRuleActionType.java

```diff
@@ -70,11 +70,16 @@
         .add( HIDEFIELD, SETMANDATORYFIELD, HIDEOPTION,
             HIDEOPTIONGROUP, SHOWOPTIONGROUP )
         .build(); // Actions associated with DataElement Or
-                  // TrackedEntityAttribute
+    // TrackedEntityAttribute
 
     private static final Set<ProgramRuleActionType> NOTIFICATION_LINKED_TYPES = new ImmutableSet.Builder<ProgramRuleActionType>()
         .add( SENDMESSAGE, SCHEDULEMESSAGE ).build(); // Actions associated with
-                                                      // NotificationTemplate
+    // NotificationTemplate
+
+    private static final Set<ProgramRuleActionType> NEW_TRACKER_TYPES = new ImmutableSet.Builder<ProgramRuleActionType>()
```

Comment:
What does "new" imply here? "New" could be old at some point, maybe better to have a descriptive name?

Reply:
ServerSupportedTypes
or
BackendSupportedTypes

Reply:
"New" would be signifying the new tracker importer!

Reply:
Refactored